spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SessionsController, type: :controller do
```

Comment:
There must be something wrong with your commits. Where did the following commit go? 
https://github.com/Abdusaid10/private-events/pull/3/commits/f80461c2c425df3d9400159deb18b05aad84f1ab